requirements.txt

```diff
@@ -1,4 +1,7 @@
 pydicom
 # This is a dcmstack branch with changes for Python 3
-git+git://github.com/ghisvail/dcmstack@master
+git+git://github.com/mvdoc/dcmstack@bf/importsys
 nipype
+inotify
```

Comment:
ideally all those should be within setup.py within corresponding groups within `requires`. and there should be a dedicated one for `monitor` with inotify and tinydb, which ppl wouldn't need if they don't use monitor.  So please move there

here in requirements we should theoretically just have
```
# To install all possibly depends as setup.py says
.[full]
# some very specific versions etc
git+git://github.com/mvdoc/dcmstack@bf/importsys
```